conf/log4j_yarn_cluster.properties

```diff
@@ -0,0 +1,23 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
why do we need to release with a new .properties file?

Reply:
Because the default log4j.properties use `DailyRollingFileAppender`, this is not suitable for yarn container.